container/example-config/basic.toml

```diff
@@ -0,0 +1,27 @@
+[[agent]]
+id = "test agent"
+name = "Holo Tester"
+key_file = "holo_tester.key"
+
+[[dna]]
+id = "app spec rust"
+file = "app-spec-rust.hcpkg"
+hash = "Qm328wyq38924y"
+
+[[instance]]
+id = "app spec instance"
```

Comment:
We had also talked about just using (agent_key, dna_hash) as the unique id for instances. If this separate notion of `id` is not implemented, we could take it out...

Reply:
(agent_key, dna_hash) could be used to set the id when creating instance configurations through code. That is actually what I'm intending to do. But I think its cleaner and easier to understand (especially if somebody writing a config manually) if there is an id field in the config file.